live-examples/css-examples/lists-and-counters/list-style.html

```diff
@@ -0,0 +1,50 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="list-style">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">list-style: square;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">list-style: square inside;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">list-style: none;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_five" class="language-css">list-style: lower-roman inside;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_five" class="language-css">list-style: url('/media/examples/firefox-logo.svg');</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example" class="default-example">
+            <ul id="example-element" class="transition-all">
```

Comment:
I would suggest adding some custom styling for the `#example-element` here as described in https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING.md#styling-the-example

All that is need is a fuller width, and bumping the `font-size` so that the image in the last example is clearer:

```
.default-example {
    font-size: 1.8rem;
}

#example-element {
    width: 50%;
}
```

Reply:
@schalkneethling, updates applied. I used 80% for example-element's width instead of 50% to prevent wrapping specially with little-roman example.